snapcraft/hooks/connect-plug-ovn-certificates

```diff
@@ -37,8 +37,10 @@ get_bool() {
 if [ -e "/etc/.lxd_generated" ]; then
   ovn_builtin=$(get_bool "$(snapctl get ovn.builtin)")
 
-  if test -d /etc/ovn ; then
```

Comment:
`/etc/ovn` can be symlink pointing to `/var/lib/snapd/hostfs/etc/ovn`. Since `test -d`|`[ -d ... ]` follow symlinks, that `if` will succeed if the final directory `/var/lib/snapd/hostfs/etc/ovn` exists. In this case, is it really a good idea to do a `rm -rf` of it?

I think we should first check if it's a symlink `[ -L ... ]` (which does not follow links ;) ) and if not, only then check if it's a dir.